portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/asset/assetpublisher/AssetPublisher.testcase

```diff
@@ -1005,4 +1005,45 @@ definition {
 		}
 	}
 
+	@description = "This is a test for LPS-135554. This test assert the Web Content Article Structures should appear after uncheck Search With Index configuration."
+	@priority = "4"
+	test CanWebContentArticleStructuresAppearAfterUncheckSearchWithIndexConfiguration {
+		property test.name.skip.portal.instance = "SystemSettings#CanWebContentArticleStructuresAppearAfterUncheckSearchWithIndexConfiguration";
+
+		task ("Uncheck Search with Index Configuration") {
+			ApplicationsMenu.gotoPortlet(
+				category = "Configuration",
+				panel = "Control Panel",
+				portlet = "System Settings");
+
+			SystemSettings.gotoConfiguration(
+				configurationCategory = "Assets",
+				configurationName = "Asset Publisher",
+				configurationScope = "System Scope");
+
+			FormFields.disableCheckbox(fieldName = "searchWithIndex");
+
+			SystemSettings.saveConfiguration();
+		}
+
+		task ("Assert the Web Content Article Structures selector appear.") {
+			
+			Navigator.gotoSitePage(
+				pageName = "Test Page Name",
+				siteName = "Test Site Name");
+
+			AssetPublisherPortlet.configureAssetTypePG(
+				assetSubtype = "Any",
+				selectedAsset = "Web Content Article");
```

Comment:
According to the steps, we want to set the configuration in System Settings first, _then_ add Asset Publisher to a page.  This is important because we're testing out search indexing for this case - if the order were different, then the indexing wouldn't apply to the Asset Publisher widget.  You can either add this widget to a new page on this test, after the System Settings configuration is complete, or put this test in a new testcase file that doesn't add Asset Publisher in the setup block